rmgpy/molecule/graph.pyx

```diff
@@ -1046,7 +1046,7 @@ cdef class Graph:
         return longest_cycle
         
         
-    cpdef bint isMappingValid(self, Graph other, dict mapping) except -2:
+    cpdef bint isMappingValid(self, Graph other, dict mapping, bint equivalent=True) except -2:
```

Comment:
Could you update the docstring with the new argument?

Reply:
done